lib/debug_logging/instance_notifier_modulizer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module DebugLogging
+  module InstanceNotifierModulizer
+    def self.to_mod(methods_to_notify: nil)
+      Module.new do
+        Array(methods_to_notify).each do |method_to_notify|
+          # method name must be a symbol
+          opts = method_to_notify.is_a?(Array) && method_to_notify.last.is_a?(Hash) && method_to_notify.pop
+          if method_to_notify.is_a?(Array)
+            method_to_notify = method_to_notify.first&.to_sym
+          else
+            method_to_notify.to_sym
+          end
+          event_name = "#{method_to_notify}.log"
```

Comment:
Would like to get a useful "<method><separator><class>" structure on the event name, like you can have with a standard implementation of AS::N.

See https://github.com/pboling/debug_logging/blob/master/lib/debug_logging/instance_logger_modulizer.rb#L23 and https://github.com/pboling/debug_logging/blob/12c8b9d46f01c6adad184aa308d4e1d80eb68c52/lib/debug_logging/argument_printer.rb#L23 for and idea of how this was done for the logging side of the gem.

Reply:
I think the instrumented event name can just be the `method_to_notify` suffixed with `.log` for any subscribers to attach.  

Grape Active Support Instrumentation follows a similar structure on the event name: 
https://github.com/ruby-grape/grape#active-support-instrumentation

The log subscriber will then directly receive instrumented events by subscribing to `/log/`.